.vscode/launch.json

```diff
@@ -0,0 +1,23 @@
+{
+    // IntelliSense를 사용하여 가능한 특성에 대해 알아보세요.
```

Comment:
@minias 
Could you remove these lines? 
or I can do it for you after merging this PR.

Reply:
In fact, it is recommended that IDE preferences be added to .gitignore and not included in shape management.